node/Cargo.toml

```diff
@@ -134,6 +135,7 @@ all-natives = [
 	"crab-native",
 	"darwinia-native",
 	"pangolin-native",
+	"pangoro-native",
```

Comment:
Add `"pangoro-runtime?/runtime-benchmarks"`  and `"pangoro-runtime?/try-runtime"`?